bin/run_invoke.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+# Invokes the pigeon lambda function in a python3.6 lambda runtime
+# environment. Pass an event in via stdin.
+#
+# This turns around and uses lambci/lambda:python3.6, so any of those
+# args will work, too.
+#
+# https://github.com/lambci/docker-lambda#example
+#
+# Usage: EVENT | ./bin/run_invoke.sh
+
+# Note: Need to set DOCKER_LAMBDA_USE_STDIN to pipe events from stdin
+docker-compose run \
+               --rm \
+               -v "$PWD/build":/var/task \
+               --service-ports \
+               -e DOCKER_LAMBDA_USE_STDIN=1 \
+               lambda-run pigeon.handler $@
```

Comment:
Honestly, I think this is the secret sauce of the whole thing. The `lambci/docker-lambda` project is pretty fab. This lets us run our function in a for-realz AWS Lambda Python 3.6 runtime locally. Pretty cool.